source/org/zfin/db/postGmakePostloaddb/1141/ZFIN-8426-construct-cleanup-for-GFP.sql

```diff
@@ -0,0 +1,87 @@
+--liquibase formatted sql
+--changeset rtaylor:ZFIN-8426-construct-cleanup-for-GFP
+
+-- delete all the marker_relationship rows that correspond to the constructs that have been re-assigned to sfGFP instead of GFP
+DELETE FROM marker_relationship WHERE "mrel_type" = 'coding sequence of' AND "mrel_mrkr_1_zdb_id" = 'ZDB-TGCONSTRCT-200901-8' AND "mrel_mrkr_2_zdb_id" = 'ZDB-EFG-070117-2' AND "mrel_comments" IS NULL AND "mrel_zdb_id" = 'ZDB-MREL-200901-31';
```

Comment:
Can these delete statements be written in a single QL query? Can you not find the marker_relationship records from finding the constructs in question by name or something else?

Reply:
I changed the query. Can you check it again?